src/pages/workspace/bills/WorkspaceBillsFirstSection.js

```diff
@@ -54,11 +56,23 @@ const WorkspaceBillsFirstSection = (props) => {
                 <Text>
                     {props.translate('workspace.bills.askYourVendorsBeforeEmail')}
                     {props.user.isFromPublicDomain ? (
-                        <TouchableOpacity
-                            onPress={() => Link.openExternalLink('https://community.expensify.com/discussion/7500/how-to-pay-your-company-bills-in-expensify/')}
-                        >
-                            <Text style={[styles.textBlue]}>example.com@expensify.cash</Text>
-                        </TouchableOpacity>
+                        <>
+                            {
+                                Platform.OS === 'android' || Platform.OS === 'ios' ? (
+                                    <TouchableNativeFeedback
+                                        onPress={() => Link.openExternalLink('https://community.expensify.com/discussion/7500/how-to-pay-your-company-bills-in-expensify/')}
+                                    >
+                                        <Text style={[styles.textBlue]}>example.com@expensify.cash</Text>
+                                    </TouchableNativeFeedback>
+                                ) : (
+                                    <TouchableOpacity
+                                        onPress={() => Link.openExternalLink('https://community.expensify.com/discussion/7500/how-to-pay-your-company-bills-in-expensify/')}
+                                    >
+                                        <Text style={[styles.textBlue]}>example.com@expensify.cash</Text>
+                                    </TouchableOpacity>
+                                )
+                            }
+                        </>
```

Comment:
Looking more at the code, I think we should use `TextLink` here.

Check the other links on the app, for example on Connect bank account .
![image](https://user-images.githubusercontent.com/24370807/165746191-c47fdeaa-9a89-41d7-8863-c6d312cc1a51.png)

Reply:
More on SignIn page. 
![image](https://user-images.githubusercontent.com/24370807/165746354-de00d354-7807-4916-a4fc-e451998d79e0.png)

Reply:
There we votes on slack as I said earlier. Your solution was proposed by someone and it was rejected, i can't really tell why. I suggest reading the Proposals in the [issue](https://github.com/Expensify/App/issues/8664).

Reply:
cc: @michaelhaxhiu

Reply:
Could you please link me to the slack thread?

Reply:
Hi @Justicea83! I'm not seeing a proposal on the issue for using a `<textLink/>` component specifically, only proposals with `<text>` or `<touchableOpacity>`. Am I just missing it somewhere?

Reply:
You're right, there was no `<textLink/>` in the proposals, but as we had `<touchOpacity/>`, my guts told me to continue with it, at least its tried and tested. I'll try the `<textLink/>` to see what it offers.

Reply:
Sweet, thanks! LMK how it goes